Oracle/PCORNetLoader_ora.sql

```diff
@@ -1335,7 +1275,8 @@ m.start_date SPECIMEN_DATE,
 to_char(m.start_date,'HH:MI')  SPECIMEN_TIME,
 m.end_date RESULT_DATE,
 to_char(m.end_date,'HH:MI') RESULT_TIME,
-CASE WHEN m.ValType_Cd='T' THEN NVL(nullif(m.TVal_Char,''),'NI') ELSE 'NI' END RESULT_QUAL, -- TODO: Should be a standardized value
+--CASE WHEN m.ValType_Cd='T' THEN NVL(nullif(m.TVal_Char,''),'NI') ELSE 'NI' END RESULT_QUAL, -- TODO: Should be a standardized value
+'NI' RESULT_QUAL, -- Temporary fix for KUMC

```

Comment:
@njgraham:  The TODO on the original line indicated that they knew that this was an issue.  After talking with Dan, he suggested this temporary fix and logging a bug with the SCHILS folks (which I did https://github.com/SCILHS/i2p-transform/issues/19).  Jeff responded on Thursday, but it sounds like they haven't found the right solution yet.